index.html

```diff
@@ -61,22 +64,146 @@ <h2>Software Developer</h2>
+            <li><a href="#"><img src="./img/twitter.png" alt="twitter"></a>
+            <li><a href="#"><img src="./img/icon-4.png" alt="Not known"></a></li>
+        </ul>
+        
+        <div class="next-page">
+            <ul class="down-key">
+                <li><img src="./img/down_icon.png" alt="Down Icon"></li>
+                <li><span>works</span></li>
             </ul>
         </div>
+
+    </section>
+    
+    
+    <div class="page-header">
+        <h2>My Recent Works</h2>
     </div>
+
+    <div class="break">
+        <img src="./img/hor_line.png" alt="Horizontal Line" id="last-lin">
+    </div>
+    
+    <section class="works">
+
+        <div class="top-work-shine-1">
+            <img src="./img/top_works_shine_1.png" alt="Shine circle">
+        </div>
+        
+        <div class="project-img">
+            <img src="./img/img_placeholder.png" alt="Image Place holder">
+        </div>
+
+        <div class="project-title">
+            <h2>
+                Multi-Post Stories
+            </h2>
+        </div>
+        
+        <div class="project-description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+
+        <ul class="tech-list">
+            <li>CSS</li>
+            <li><img src="./img/vertical_col.png" alt="vertical_col"></li>
+            <li>HTML</li>
+            <li><img src="./img/vertical_col.png" alt="vertical_col"></li>
+            <li>Bootstrap</li>
+            <li><img src="./img/vertical_col.png" alt="vertical_col"></li>
+            <li>Ruby</li>
+        </ul>
+
+        <div class="see-more">
+            <button type="submit">See Project</button>
+        </div>
+    </section>
+
+    <section class="works">
+
+        <div class="bootom-work-shine-2">
+            <img src="./img/top_works_shine_1.png" alt="Shine circle">
+        </div>
+        
+        <div class="project-img">
+            <img src="./img/img_placeholder.png" alt="Image Place holder">
+        </div>
+
+        <div class="project-title">
+            <h2>
+                Multi-Post Stories
+            </h2>
+        </div>
+        
+        <div class="project-description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+
+        <ul class="tech-list">
+            <li>CSS</li>
+            <li><img src="./img/vertical_col.png" alt="vertical_col"></li>
+            <li>HTML</li>
+            <li><img src="./img/vertical_col.png" alt="vertical_col"></li>
+            <li>Bootstrap</li>
+            <li><img src="./img/vertical_col.png" alt="vertical_col"></li>
+            <li>Ruby</li>
+        </ul>
```

Comment:
- [ ] Kindly fix the extra space in your code as it's making the UI (User Interface) poor 😌
- Your page
![screenshot (40)](https://user-images.githubusercontent.com/44624138/149630271-9758bf78-e3a0-4a89-9eee-3b02be210cec.png)